server/.env

```diff
@@ -7,4 +7,7 @@ BASE_URL=http://localhost:3000
 # Prisma supports the native connection string format for PostgreSQL, MySQL, SQLite, SQL Server, MongoDB and CockroachDB.
 # See the documentation for all the connection string options: https://pris.ly/d/connection-strings
 
-DATABASE_URL="postgresql://postgres:postgres@localhost:5432/postgres?schema=public"
\ No newline at end of file
+DATABASE_URL="postgresql://postgres:postgres@localhost:5432/postgres?schema=public"
+
+EMAIL_USER= "17ict063@seu.ac.lk"
```

Comment:
I'm yet to test your code, but if this approach works for you, can we get a temporary gmail account created for the app? You will have to turn on 'less secure' option in gmail settings, and get an `app password` created and use that here. Otherwise Gmail authentication will fail.